README.adoc

```diff
@@ -1,48 +1,166 @@
   Example `mvn quarkus:dev -Dquarkus.args="https://github.com/user/repo.git"`
-- Building native executable: `mvnw package -Dnative`
+- **Building native executable** : `mvnw package -Dnative`
 
-## Useful resources:
-As Starfish uses Command Mode in Quarkus refer these links to learn more about it:
+=== Links/Resources:
+As Starfix uses Command Mode in Quarkus refer these links to learn more about it:
 
 - https://quarkus.io/blog/introducing-command-mode/[Introducing Command Mode ] written by https://github.com/maxandersen[@maxandersen]
 - https://quarkus.io/guides/command-mode-reference[QUARKUS - COMMAND MODE APPLICATIONS]
 
-## Contributing to Starfish
-- Issues: Start by looking at the existing issues at https://github.com/maxandersen/starfish/issues[starfish/issues] and create a Pull Request with appropriate description/comment.
-- Bugs: Run the starfish application and if you find unexpected behaviour then open a new issue with label `bug` and include a proper description/comment.
-- Feature request: Brainstorm ideas and bring them to surface on the features that can be implemented in starfish and then create an issue with label as `enahancement`.Always include as exhaustive description as possible to help understand better.
 
-## Technology for Development
-- https://quarkus.io/[Quarkus]
+
+== Technology for Development
+- https://quarkus.io/[Quarkus] : The Starfish CLI is built as a native executable with Quarkus.Specifically,we are using the **command mode** of Quarkus.
+- https://junit.org/junit5/[JUNIT5]:  As Quarkus supports JUNIT5, it has been used for Unit Testing.
+- https://maven.apache.org/plugins/maven-assembly-plugin/[Maven Assembly] : As we needed to combine project output(native exectable) into a single distributable archive that along with installer scripts  we are using Maven Assembly.
+- https://github.com/features/actions[Github Actions] : Github actions makes our CI/CD very easy.We are using it build,test and generate artifacts and distributable packages.Currently we are using it  on Pull Request(Builds and generates artifacts), Push(Builds and generates artifacts) and  Releases(generates distributable packages as zip file).We have also set-up a release mechanism on pushing with tags v* .
+
+== Contributing to Starfix
+- Issues: Start by looking at the existing issues at https://github.com/maxandersen/starfish/issues[starfix/issues] and create a Pull Request with appropriate description/comment.
+- Bugs: Run the starfix application and if you find unexpected behaviour then open a new issue with label `bug` and include a proper description/comment.
+- Feature request: Brainstorm ideas and bring them to surface on the features that can be implemented in starfix and then create an issue with label as `enahancement`.Always include as exhaustive description as possible to help understand better.
```

Comment:
```suggestion
- Feature request: Brainstorm ideas and bring them to surface on the features that can be implemented in `starfix` and then create an issue with label as `enhancement`. Always include as exhaustive description as possible to help understand better.
```

I notice you often write without spacing after a `.` - best to follow standard english punctuation; thus it is with a space after the `.`. Like that <-- ;)